shared/types/payment.ts

```diff
@@ -18,6 +18,12 @@ export enum PaymentChannel {
   Stripe = 'Stripe',
   // for extensibility to future payment options
 }
+
+export enum PaymentMethodType {
+  Unset = '',
```

Comment:
can add a comment explaining what this means?

Reply:
Good catch. This is not intended to be commit in. I'll remove it!

Reply:
0dee01943